generators/cypress/templates/src/test/javascript/cypress/support/keycloak-oauth2.ts.ejs

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable @typescript-eslint/no-namespace */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+
+import { getAuthCodeFromLocation } from './utils';
+
+Cypress.Commands.add("kcLogin", (user: string) => {
```

Comment:
I have some trouble to understand these: does the cy.login method works with keyckloak, or should it be repolaced by kcLogin ?

Reply:
kcLogin replace cy.login in case of oauth2 mode.